composer.json

```diff
@@ -15,17 +15,17 @@
     }
   },
   "require": {
-    "php": ">=5.6"
+    "php": "^7.1"
```

Comment:
```suggestion
    "php": ">=7.1"
```

this needs to be >= to avoid ticking time bomb

Reply:
done

Reply:
Haha speaking of that: here's proof why ^7.1 is troublesome: https://travis-ci.com/github/roots/wp-config/jobs/297823698#L217

One of our deps is doing that